packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -21,6 +21,7 @@ const authStatusElm = document.querySelector('#access-token-status');
 const registerElm = document.querySelector('#registration-register');
 const unregisterElm = document.querySelector('#registration-unregister');
 const registrationStatusElm = document.querySelector('#registration-status');
+const productionEnv = document.getElementById('production');
```

Comment:
is this needed?

Reply:
ya want to disable it and dont want to refer it twice